templates/nro/.circleci/config.yml.tmpl

```diff
@@ -430,105 +412,91 @@ workflows:
   develop:
     jobs:
       - visualtests-reference-develop:
-          <<: *develop_common
+          <<: *on_develop_commit
       - build-develop:
-          <<: *develop_common
+          <<: *on_develop_commit
       - deploy-develop:
-          <<: *develop_common
+          <<: *on_develop_commit
           requires:
             - build-develop
             - visualtests-reference-develop
 {{- if isTrue .Env.MAKE_RELEASE }}
             - merge-develop
 {{- end }}
       - test-develop:
-          <<: *develop_common
+          <<: *on_develop_commit
           requires:
             - deploy-develop
       - visualtests-compare-develop:
-          <<: *develop_common
+          <<: *on_develop_commit
           requires:
             - deploy-develop
 {{- if isTrue .Env.MAKE_RELEASE }}
       - merge-develop:
-          <<: *develop_common
+          <<: *on_develop_commit
 {{- else }}
       - data-sync:
-          <<: *develop_common
+          <<: *on_develop_commit
           requires:
             - deploy-develop
 {{- end }}
 
 {{- if isTrue .Env.MAKE_RELEASE }}
-  staging:
-    unless:
-      not:
-        equal: [ "staging", << pipeline.parameters.release_stage >> ]
+  release:
+    when:
+      equal: [ "release", << pipeline.parameters.release_stage >> ]
     jobs:
       - visualtests-reference:
-          <<: *staging_common
-      - build-staging:
-          <<: *staging_common
+          <<: *on_release_tag
```

Comment:
Since this definition is added to each job for both develop and release workflows, maybe there's a way in CircleCI to write it once on a workflow level?